RecoEgamma/EgammaElectronProducers/python/gsfElectronSequence_cff.py

```diff
@@ -19,8 +20,89 @@
 
 #gsfElectronMergingSequence = cms.Sequence(gsfElectronCores*gsfElectrons)
 
-from RecoEgamma.EgammaElectronProducers.edBasedElectronIso_cff import *
 from RecoEgamma.EgammaElectronProducers.pfBasedElectronIso_cff import *
+ 
+from CommonTools.ParticleFlow.pfParticleSelection_cff import *
+from RecoEgamma.EgammaElectronProducers.electronPFIsolationDeposits_cff import *
+
+elEDIsoDepositCharged=elPFIsoDepositCharged.clone()
+elEDIsoDepositCharged.src="ecalDrivenGsfElectrons"
```

Comment:
what is the purpose of exploding this file in size and merging the content defined in  RecoEgamma/EgammaElectronProducers/python/electronEDIsolationDeposits_cff.py and  RecoEgamma/EgammaElectronProducers/python/electronEDIsolationValues_cff.py

The isolation defs are fairly self-contained and could stay in their corresponding files, exposing just the final sequence needed to build the electronIsoTask.
Overall the purpose of these isolation modules/tasks in gsfElectonSequence_cff is fairly nebulous.
I couldn't find valid usecases for either electronIsoSequence or gsfElectronMergingSequence

Reply:
Hi @slava77, thanks for the comments! You are right, here it should have stayed in a separate file because it is rather self-contained as you say. Putting the isolation definitions in that file was my way of making explicit that they are only used in gsfElectonSequence_cff, but now that the tests passed and we had a first look at it I'll revisit.